spring-boot-project/spring-boot-test/src/main/java/org/springframework/boot/test/web/client/MockServerRestTemplateCustomizer.java

```diff
@@ -96,13 +101,25 @@ public void setDetectRootUri(boolean detectRootUri) {
 		this.detectRootUri = detectRootUri;
 	}
 
+	/**
+	 * Use the {@link BufferingClientHttpRequestFactory} wrapper to buffer the input and
+	 * output streams, and for example, allow multiple reads of the response body.
+	 */
+	public void bufferContent() {
```

Comment:
I think this should be a setter that takes a `boolean`, aligning it with root URI detection.

Reply:
Done :+1: